modules/ROOT/pages/mobile-developer-console.adoc

```diff
@@ -2,27 +2,21 @@
 include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 
-= {mobile-client}s in OpenShift
+= Mobile Developer Console
 :toc:
 
 == Introduction
 
-{mobile-client}s in OpenShift allow you to perform Cloud Native Mobile App Development using OpenShift as the back-end
```

Comment:
I think we still need at least some of this text.
but I'm not sure if it fitx under this new heading

We have to action for the user with this heading, and all you could expect is that we explain what the console is, a few lines....( we never explained 'OpenShift Console')

With the old heading, we were introducing the concept that is required for mobile services, the idea that a representation of your mobile app exists in OS and can be bound to services, with a json file used to communicate that configuration to your local dev environment, don't we need a heading for that fundamental concept? I remember it was the only thing that JF asked me to add to content plan ('the inception moment') @laurafitzgerald

Reply:
@finp do we capture that concept in the getting-started guide at any stage. Could we reuse it here?